packages/http-client-java/eng/scripts/Build-Packages.ps1

```diff
@@ -74,6 +74,9 @@ try {
 
     Invoke-LoggedCommand "npm run build" -GroupOutput
 
+    # linting
+    Invoke-LoggedCommand "npm run build" -GroupOutput
```

Comment:
@haolingdong-msft `build`?

Reply:
also there are some not found by lint

```
C:\github\autorest.java\typespec-extension\src\code-model-builder.ts
  2344:18  warning  'details' is deprecated. Use `doc` and `summary` instead  deprecation/deprecation

C:\github\autorest.java\typespec-extension\src\external-schemas.ts
  285:23  warning  'description' is deprecated. Use `doc` and `summary` instead  deprecation/deprecation
  286:49  warning  'description' is deprecated. Use `doc` and `summary` instead  deprecation/deprecation
  288:23  warning  'details' is deprecated. Use `doc` and `summary` instead      deprecation/deprecation
  289:70  warning  'details' is deprecated. Use `doc` and `summary` instead      deprecation/deprecation
```

Reply:
Oops. thanks for the catch! Yeah, autorest.java has lint errors.
I will fix it in another pr.